auxprogs/homGeneMapping/src/Makefile

```diff
@@ -36,10 +36,10 @@ all: homGeneMapping
 .SUFFIXES: .cc .o .so
 
 .cc.o:
-	$(CC) -c $(CFLAGS) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $< $(INCLS)
+	$(CXX) -c $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $< $(INCLS)
 
 homGeneMapping: main.cc $(OBJS)
-	$(CC) $(CFLAGS) $(CPPFLAGS) -o $@ $^ $(INCLS)  $(LIBS)
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@ $^ $(INCLS) $(LIBS)
 	mkdir -p ../../../bin/
 	cp homGeneMapping ../../../bin/homGeneMapping
 
```

Comment:
hmehlan has additionally changed this file:
- L10: in this file macro CC is defined as g++ -> better use CXX and make it overidable (as in auxprogs/joingenes/Makefile src/Makefile)
- L15,22 CFLAGS/CPPFLAGS should renamed to CXXFLAGS as it contains flags for the C++ compiler (as in auxprogs/joingenes/Makefile)
- L39,42 later CFLAGS should no longer used but CXXFLAGS (if not already there)
- L15 typo: CPPCLAGS (in first commit)
-L15 why -O2 is removed? but in src/Makefile -O3 stays

satta had previously changed L15:
-CFLAGS := -Wall -Wno-sign-compare -ansi -pedantic -std=c++0x -pthread -O2 ${CFLAGS} # -DDEBUG -g -ggdb -pg
+CPPFLAGS := -Wall -Wno-sign-compare -ansi -pedantic -std=c++0x -pthread ${CPPCLAGS}

Sascha: it is ok as it is now?